README.md

```diff
@@ -1,16 +1,42 @@
 # Elasticsearch
 
-This repository contains Ruby integrations for [Elasticsearch](https://www.elastic.co/products/elasticsearch).
+This repository contains the official [Elasticsearch](https://www.elastic.co/products/elasticsearch) Ruby client.
 
 [![6.x](https://github.com/elastic/elasticsearch-ruby/actions/workflows/6.x.yml/badge.svg?branch=6.x)](https://github.com/elastic/elasticsearch-ruby/actions/workflows/6.x.yml) [![7.x](https://github.com/elastic/elasticsearch-ruby/workflows/7.x/badge.svg?branch=7.x)](https://github.com/elastic/elasticsearch-ruby/actions/workflows/7.x.yml) [![main](https://github.com/elastic/elasticsearch-ruby/actions/workflows/main.yml/badge.svg?branch=main)](https://github.com/elastic/elasticsearch-ruby/actions/workflows/main.yml) [![Code Climate](https://codeclimate.com/github/elastic/elasticsearch-ruby/badges/gpa.svg)](https://codeclimate.com/github/elastic/elasticsearch-ruby)
 
-The [`elasticsearch`](https://github.com/elasticsearch/elasticsearch-ruby/tree/main/elasticsearch) library is a wrapper for two separate libraries:
+The [`elasticsearch`](https://github.com/elasticsearch/elasticsearch-ruby/tree/main/elasticsearch) gem is a complete Elasticsearch client which uses two separate libraries:
 
-* [`elastic-transport`](https://github.com/elastic/elastic-transport-ruby), which provides a low-level Ruby client for connecting to an Elasticsearch cluster
-* [`elasticsearch-api`](https://github.com/elasticsearch/elasticsearch-ruby/tree/main/elasticsearch-api), which provides a Ruby API for the Elasticsearch RESTful API
+* [`elastic-transport`](https://github.com/elastic/elastic-transport-ruby) - provides the low-level code for connecting to an Elasticsearch cluster.
+* [`elasticsearch-api`](https://github.com/elasticsearch/elasticsearch-ruby/tree/main/elasticsearch-api) - provides a Ruby API for the Elasticsearch RESTful API.
 
-Both of these libraries are extensively documented.
-**Please read the [`elastic-transport`](https://rubydoc.info/github/elastic/elastic-transport-ruby/) and the [`elasticsearch-api`](http://rubydoc.info/gems/elasticsearch-api) documentation carefully.**
+Both of these libraries are extensively documented. **Please read the [`elastic-transport`](https://rubydoc.info/github/elastic/elastic-transport-ruby/) and the [`elasticsearch-api`](http://rubydoc.info/gems/elasticsearch-api) documentation carefully.**
+
+You can find more in-depth documentation in the Elastic page for the [Elasticsearch Ruby Client](https://www.elastic.co/guide/en/elasticsearch/client/ruby-api/current/index.html).
+
+## Installation
+
+Install the `elasticsearch` gem from [Rubygems](https://rubygems.org/gems/elasticsearch):
+
+```
+$ gem install elasticsearch
+```
+
+Or add it to your project's Gemfile:
+
+```ruby
+gem 'elasticsearch', 'VERSION'
+```
+
+To install from a source code checkout:
```

Comment:
Would we typically be able to support someone who did this in a production context? If not, should we add clarification here?

Reply:
I'll remove that since it's not something we'd support and is probably very specific to developers who would know how to do this anyway.